pages/common/whoami.md

```diff
@@ -1,7 +1,11 @@
 # whoami
 
-> Show the username of the current user.
+> Print the username associated with the current effective user ID.
 
-- Display currently logged user name:
+- Display currently logged username:
 
 `whoami`
+
+- Display the username after a change in the user ID:
```

Comment:
Does saying "Display the root username help" ?

Reply:
I think uid 0 is always printed as root.

Reply:
Ah ok.